libs/editor/egui_editor/src/appearance.rs

```diff
@@ -76,6 +77,7 @@ pub struct Appearance {
     pub checkbox_slash_width: Option<f32>,
     pub rule_height: Option<f32>,
     pub image_padding: Option<f32>,
+    pub base_font_size: Option<f32>,
```

Comment:
the interface lets you set a font size in absolute terms so you can coordinate with client font size